### PR TITLE
Testloader support for custom paths.

### PR DESCRIPTION
**Pull request by GeisaFaustino**
Add optional parameters to make possible provide custom paths for testing purposes.
- add two optional parameters on `TestSectionLoader` for providing custom seismic and label paths;
- add one optional parameter on `TestPatchLoader` to make possible provide custom *.txt path.

In both cases, the default option is not providing custom paths.

### CODE REVIEW

File: interpretation/deepseismic_interpretation/dutchf3/data.py [hunk 1]
```diff
@@ -447,7 +447,8 @@ def __getitem__(self, index):
 
 
 class TestSectionLoader(SectionLoader):
-    def __init__(self, data_dir, split="test1", is_transform=True, augmentations=None):
+    def __init__(self, data_dir, split="test1", is_transform=True, augmentations=None,
+                 seismic_path=None, label_path=None):
```
Comment (maxkazmsft) [inline interpretation/deepseismic_interpretation/dutchf3/data.py]: Please add documentation regarding the parameters

File: interpretation/deepseismic_interpretation/dutchf3/data.py [hunk 2]
```diff
@@ -576,7 +581,8 @@ def transform(self, img, lbl):
 
 
 class TestPatchLoader(PatchLoader):
-    def __init__(self, data_dir, stride=30, patch_size=99, is_transform=True, augmentations=None):
+    def __init__(self, data_dir, stride=30, patch_size=99, is_transform=True, augmentations=None,
+                 txt_path=None):
```
Comment (maxkazmsft) [inline interpretation/deepseismic_interpretation/dutchf3/data.py]: Similar comment regarding documenting the arguments

**Comment by GeisaFaustino** [comment]
Waiting for changes on data.py from TrainLoader_update branch of @MJZawacki.

**Comment by GeisaFaustino** [comment]
This PR will be abandoned/closed and modifications to provide support for custom paths regarding tests will be available soon in other branch.

**Pull request closed without merging**